csfieldguide/chapters/content/en/algorithms/sections/whats-the-big-picture.md

```diff
@@ -122,7 +122,7 @@ Some algorithms will always make the same number of comparisons for a certain in
 If we develop or are given an algorithm to solve a problem, how do we know that it works?
 Sometimes we create test cases to verify the algorithm produces correct output for specific input values.
 While this is a useful practice and can help verify that we are on the right track, it is not enough to show that our algorithm is correct.
-The old adage "even a broken watch is correct twice a day" is a good analogy.
+The old {glossary-link term="adage"}adage{glossary-link end} "even a broken watch is correct twice a day" is a good analogy.
```

Comment:
I don't think we should provide a definition of an uncommon word, as you would be breaking the flow of reading for most students. The glossary entries should be extra helpful information, not required reading.

This sentence would be much simplier if the word was changed to 'saying'.

```suggestion
The old saying "even a broken watch is correct twice a day" is a good analogy.
```